gulpfile.js

```diff
@@ -512,12 +512,12 @@ function injectARMCache(flavor, done) {
             fs.mkdirSync(linux32Folder);
         }
         const downloadedArchivePath = `${versionFolder}/nwjs${flavorPostfix}-v${nwArmVersion}-linux-arm.tar.gz`;
-        const downloadUrl = `https://github.com/LeonardLaszlo/nw.js-armv7-binaries/releases/download/v${nwArmVersion}/nwjs${flavorDownloadPostfix}-v${nwArmVersion}-linux-arm.tar.gz`;
+        const downloadUrl = 'https://github.com/LeonardLaszlo/nw.js-armv7-binaries/releases/download/nw60-arm64_2022-01-08/nw60-arm64_2022-01-08.tar.gz';
```

Comment:
Hardcoding SDK URLs deep down in the build code is almost never a good idea, especially if prospecting to have multiple builds branch out of this one. I'd keep the modular URL approach even if needs to be simplified to a single global var.